### PR TITLE
Update Key manager guides with the new testnet

### PR DESCRIPTION
**Pull request by b00ste**
### What does this PR introduce?

The changes update the code to work with the new LUKSO Testnet.
Additionally **Execute Relay Transactions** & **Upgrade LSP6 Key Manager** have been tested.
**Get controller addresses** & **Grant Permissions to 3rd parties** have to be tested after _erc725.js_ repository is updated to the latest version of _lsp-smart-contracts_

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [CU-85zt0y76b Update Key Manager Guides to the new Testnet](https://app.clickup.com/t/85zt0y76b)

**Review by YamenMerhi — APPROVED** [review]

File: docs/guides/key-manager/execute-relay-transactions.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ const keyManager = new web3.eth.Contract(KeyManagerContract.abi, keyManagerAddre
 ```typescript title="Contract instances"
 const universalProfile = new ethers.Contract(universalProfileAddress, UniversalProfileContract.abi);
 
-const keyManagerAddress = await universalProfile.owner();
+const keyManagerAddress = await universalProfile.connect(controllerAccount).owner();
```
Comment (CJ42) [inline docs/guides/key-manager/execute-relay-transactions.md:133]: is the `.connect(...)` needed here? This is just a getter function. Should not be necessary.

File: docs/guides/key-manager/execute-relay-transactions.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ const nonce = await keyManager.methods.getNonce(controllerAccount.address, chann
 
 ```typescript title="Get the controller key nonce"
 const channelId = 0;
-const nonce = await keyManager.getNonce(controllerAccount.address, channelId);
+const nonce = await keyManager.connect(controllerAccount).getNonce(controllerAccount.address, channelId);
```
Comment (CJ42) [inline docs/guides/key-manager/execute-relay-transactions.md:168]: same here.

File: docs/guides/key-manager/execute-relay-transactions.md [hunk 3]
```diff
@@ -201,7 +201,7 @@ const abiPayload = universalProfile.interface.encodeFunctionData(
   [
     0, // Operation type: CALL
     '0x...', // Recipient address
-    web3.utils.toWei('1'), // Value
+    ethers.utils.parseUnits("1", "ether"), // Value
```
Comment (CJ42) [inline docs/guides/key-manager/execute-relay-transactions.md:204]: good catch 👌

**Review by CJ42 — CHANGES_REQUESTED** [review]

File: docs/guides/key-manager/execute-relay-transactions.md (hunk 1 above)
Comment (b00ste) [reply docs/guides/key-manager/execute-relay-transactions.md:133]: getting error without because it doesn't know the provider

**Review by CJ42 — APPROVED** [review]

**Pull request merged**